extensions/amp-sidebar/1.0/sidebar-config.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {useRef} from '../../../src/preact';
+
+/** @protected @enum {string} */
+export const Side = {
+  LEFT: 'left',
+  RIGHT: 'right',
+};
+
+/**
+ * @param {T} current
+ * @return {{current: T}}
+ * @template T
+ */
+export function useValueRef(current) {
```

Comment:
This is not a config. Let's just go ahead and put this into `/src/preact/component/value-ref.js` and export it via `/src/preact/component/index.js`.

Reply:
Updated in latest.